frontend/src/app/app-routing/app-routing.module.ts

```diff
@@ -2,12 +2,14 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import { SignInComponent } from '../sign-in/sign-in.component';
+import { AuthGuard } from '../auth/auth.guard';
 
 const routes: Routes = [
-  {
-    path: 'sign-in',
-    component: SignInComponent
-  }
+  { path: 'sign-in', component: SignInComponent, canActivate: [AuthGuard] },
+  { path: 'party', component: SignInComponent, canActivate: [AuthGuard] },
```

Comment:
path `party` should not be routed to `SignInComponent`

Reply:
Since the component except `SignInComponent` doesn't exist yet, my intention was to mock the component first (to `SignInComponent`, the one and only, cute and little component) and then change it to the specified component later.